frontend/src/pages/projects/screens/spawner/dataConnection/DataConnectionField.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import { Checkbox, FormGroup, Radio, Stack, StackItem } from '@patternfly/react-core';
+import {
+  DataConnectionData,
+  EnvironmentVariableType,
+  SecretCategory,
+} from '~/pages/projects/types';
+import AWSField from '~/pages/projects/dataConnections/AWSField';
+import { EMPTY_AWS_SECRET_DATA } from '~/pages/projects/dataConnections/const';
+import ExistingDataConnectionField from './ExistingDataConnectionField';
+
+type DataConnectionFieldProps = {
+  dataConnection: DataConnectionData;
+  setDataConnection: (dataConnection: DataConnectionData) => void;
+};
+const DataConnectionField: React.FC<DataConnectionFieldProps> = ({
+  dataConnection: dataConnection,
+  setDataConnection: setDataConnection,
```

Comment:
Isn't this syntactically erroneous?

Reply:
what do you mean? the prop types?

Reply:
Unless I am missing something -- are you not pulling out of props `dataConnection` and then renaming it to `dataConnection`?

Reply:
I didn't run this in my IDE, so it's possible I'm misreading the code lol

Reply:
oh i see. im renaming it to the same thing.